runtime/jcl/common/mgmtosext.c

```diff
@@ -79,6 +79,33 @@ Java_com_ibm_lang_management_internal_ExtendedOperatingSystemMXBeanImpl_getProce
 	return (0 == rc)? (jlong) size: (jlong) -1;
 }
 
+jdouble JNICALL
+Java_com_ibm_lang_management_internal_ExtendedOperatingSystemMXBeanImpl_getSystemCpuLoadImpl(JNIEnv *env, jobject instance) {
+	PORT_ACCESS_FROM_ENV(env);
+   OMRPORT_ACCESS_FROM_J9PORT(portLibrary);
```

Comment:
should be [`PORTLIB`](https://github.com/eclipse/openj9/blob/master/runtime/oti/j9port.h#L56), I think `portLibrary` won't be defined.

Reply:
Fixed in e200fcd